src/identity-providers/add/DetailSettings.tsx

```diff
@@ -140,6 +159,35 @@ export const DetailSettings = () => {
   const isOIDC = id.includes("oidc");
   const isSAML = id.includes("saml");
 
+  const loader = async () => {
+    const loaderMappers = await adminClient.identityProviders.findMappers({
+      alias: id,
+    });
+
+    const loaderMapperTypes =
+      await adminClient.identityProviders.findMapperTypes({ alias: id });
+
+    const components = (loaderMappers as IdPWithMapperAttributes[])?.map(
+      (loaderMapper: IdPWithMapperAttributes) => {
+        const provider = Object.values(loaderMapperTypes)?.filter(
+          (loaderMapperType) => {
+            return (
+              loaderMapper.identityProviderMapper! === loaderMapperType?.id!
```

Comment:
The `loaderMapperType` variable is always defined so there is no need to optional chain this.
```suggestion
              loaderMapper.identityProviderMapper! === loaderMapperType.id!
```